docs/tutorials/single-dev-workflow.md

```diff
@@ -0,0 +1,206 @@
+---
+sidebar_label: "Epinio Journey: Single Dev Workflow"
+title: ""
+---
+
+## Introduction
+
+While the [quickstart](./quickstart.md) is great to get started, as a Developer, you might want to see end-to-end solutions and how Epinio can help you.
+That's exactly the aim of the "Epinio Journeys", where you'll be able to follow different use-cases, according to your needs.
+
+In this particular tutorial, we focus on the single dev workflow and go through a full example from Epinio installation to your first app deployment.
+
+## Prerequisites
+
+Before you can use Epinio, you need to have a Kubernetes cluster running.
+
+As a single dev, you might be using a local Kubernetes cluster such as [Rancher Desktop](https://rancherdesktop.io/), [Minikube](https://minikube.sigs.k8s.io/docs/) or [k3d](https://k3d.io/), just to name a few.
+
+:::note
+
+This tutorial will not explain how to install a local Kubernetes cluster and assumes you know how to do it.
+However, depending on your choice, there's already some examples in the **Epinio Howtos** section which contains links to the respective clusters' installation docs.
+
+:::
+
+In this tutorial, we'll use [Rancher Desktop](../howtos/install_epinio_on_rancher_desktop.md) as our local Kubernetes cluster. However you should be able to follow this tutorial with the local Kubernetes cluster of your choice.
+
+If not already done, you can install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/releases) of Rancher Desktop for your operating system.
+
+### [Optional] Additional binaries
+
+Two additional binaries need to be installed in your system:
```

Comment:
I think it might be good to be explicit that these additional binaries are only needed for the installation and customization. The actual dev flow doesn't need k8s tools